### PR TITLE
Mike's volume status changes

### PR DESCRIPTION
**Pull request by iancmcc**
Run 'serviced volume status' while using devicemapper to see the hotness

### CODE REVIEW

File: cli/cmd/volume.go [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+	fmt.Printf("DataSpaceUsed:          %s\n", bytefmt.ByteSize(status.DataSpaceUsed))
+	fmt.Printf("DataSpaceTotal:         %s\n", bytefmt.ByteSize(status.DataSpaceTotal))
+	fmt.Printf("MetadataFile:           %s\n", status.MetadataFile)
+	fmt.Printf("MetadataLoopback:       %s\n", status.MetadataLoopback)
+	fmt.Printf("MetadataSpaceAvailable: %s\n", bytefmt.ByteSize(status.MetadataSpaceAvailable))
+	fmt.Printf("MetadataSpaceUsed:      %s\n", bytefmt.ByteSize(status.MetadataSpaceUsed))
+	fmt.Printf("MetadataSpaceTotal:     %s\n", bytefmt.ByteSize(status.MetadataSpaceTotal))
+	fmt.Printf("SectorSize:             %s\n", bytefmt.ByteSize(status.SectorSize))
+	fmt.Printf("UdevSyncSupported:      %t\n", status.UdevSyncSupported)
+}
```
Comment (smousa) [inline cli/cmd/volume.go]: JSON print format?

File: rpc/master/hosts_client.go [hunk 2]
```diff
@@ -67,3 +69,14 @@ func (c *Client) FindHostsInPool(poolID string) ([]host.Host, error) {
 	}
 	return response, nil
 }
+
+//GetVolumeStatus gets status information for the given volume or nil
```
Comment (smousa) [inline rpc/master/hosts_client.go]: This needs to go in volume_client.go

File: rpc/master/hosts_server.go [hunk 3]
```diff
@@ -76,3 +78,15 @@ func (s *Server) FindHostsInPool(poolID string, hostReply *[]host.Host) error {
 	*hostReply = hosts
 	return nil
 }
+
+// GetVolumeStatus gets the volume status
+func (s *Server) GetVolumeStatus(volumeNames []string, reply *volume.Statuses) error {
```
Comment (smousa) [inline rpc/master/hosts_server.go]: This needs to go in volume_server.go

File: cli/cmd/volume.go [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+	if err != nil {
+		glog.Errorf("error getting volume status: %v", err)
+		return
+	}
+	if ctx.Bool("verbose") {
+		if jsonStatuses, err := json.MarshalIndent(response, " ", "  "); err != nil {
+			fmt.Fprintf(os.Stderr, "failed to marshal volume status list: %s", err)
+		} else {
+			fmt.Println(string(jsonStatuses))
+		}
```
Comment (morr-zenoss) [inline cli/cmd/volume.go]: JSON print format.

File: cli/cmd/volume.go (hunk 1 above)
Comment (morr-zenoss) [inline cli/cmd/volume.go]: See lines 58-62.

Comment (smousa) [inline cli/cmd/volume.go]: got it, thanks

**Pull request merged**